api/lib/domain/services/pole-emploi-service.js

```diff
@@ -0,0 +1,25 @@
+const settings = require('../../config');
+
+function generateLink(sending) {
+  const host = settings.poleEmploi.poleEmploiAPIManagerUrl;
+  const { dateEnvoi, idEnvoi } = sending;
+  const cursor = _generateCursor({ idEnvoi, dateEnvoi });
+  return `${host}/pole-emploi/envois?curseur=${cursor}`;
+}
+
+function _generateCursor(data) {
+  const string = JSON.stringify(data);
+  const buffer = new Buffer.from(string);
+  return buffer.toString('base64');
+}
+
+function decodeCursor(strbase64) {
+  if (strbase64) {
+    const buffer = new Buffer.from(strbase64, 'base64');
+    const string = buffer.toString('ascii');
+    return JSON.parse(string);
+  }
+  return null;
+}
```

Comment:
**suggestion:** Je propose ici le ["return early patterm"](https://dev.to/jpswade/return-early-12o5)
```suggestion
function decodeCursor(strbase64) {
  if (!strbase64) return null;
  
  const buffer = new Buffer.from(strbase64, 'base64');
  const string = buffer.toString('ascii');
  return JSON.parse(string);
}
```